runtime/src/iree/base/loop_emscripten.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This is the JavaScript side of loop_emscripten.c
+//
+// References:
+//   * https://emscripten.org/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.html
+//   * https://github.com/evanw/emscripten-library-generator
+//   * https://github.com/emscripten-core/emscripten/tree/main/src
+
+const LibraryLoopEmscripten = {
+  $loop_emscripten_support__postset: 'loop_emscripten_support();',
+  $loop_emscripten_support: function() {
+    class LoopEmscripten {
+      constructor() {
+        // TODO(scotttodd): store state here
+      }
+
+      loop_command_call(callback, user_data, loop) {
+        console.log('loop_command_call');
+
+        const iree_ok_status = 0;
```

Comment:
got confused here as `iree_ok_status` is a function in C - you probably want `IREE_STATUS_OK` to match the status codes